app/models/nonprofit.rb

```diff
@@ -276,6 +277,13 @@ def add_billing_subscription
     b_sub = build_billing_subscription(billing_plan: billing_plan, status: 'active')
   end
 
+  def add_scheme_to_website
+    website = self.website
+    if website.present? && (website =~ URI.regexp(['https', 'http'])).blank?
```

Comment:
What happens if they have an non `https` or `http` scheme? I think this should only modify it if the scheme is missing.

Reply:
Fixed!